packages/sign_in_with_apple/lib/src/sign_in_with_apple.dart

```diff
@@ -0,0 +1,215 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
+import 'package:meta/meta.dart';
+import 'package:flutter/services.dart';
+
+import 'package:sign_in_with_apple/sign_in_with_apple.dart';
+
+// Mention all internal imports implicitly, so we don't forget to export required classes
+import 'package:sign_in_with_apple/src/authorization_credential.dart'
+    show
+        parseAuthorizationCredentialAppleID,
+        parseAuthorizationCredentialAppleIDFromDeeplink,
+        parseAuthorizationCredentialPassword;
+import 'package:sign_in_with_apple/src/credential_state.dart'
+    show parseCredentialState;
+
+/// Wrapper class providing the methods to interact with Sign in with Apple.
+// ignore: avoid_classes_with_only_static_members
+class SignInWithApple {
```

Comment:
@HenriBeck Thought about dropping the wrapping class entirely, but since it has 4 exported methods, that also seems messy for the consumer (as each function then should be more descriptive. A plain `isAvailable` would be confusing…)

Reply:
True, if we would drop the class, we would need to make the function names significantly longer